### ISSUE TITLE
Regular Expression Denial of Service

### ISSUE THREAD

**Issue by HDVinnie** [JeffreyWay/laravel-mix#2341]
```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ acorn                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=7.1.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ laravel-mix [dev]                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ laravel-mix > webpack > acorn                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1488                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 moderate severity vulnerability in 18240 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by swingywc** [comment]
Encountering the same problem on my side.
I have seen a PR for fixing this issue in `webpack` package: https://github.com/webpack/webpack/pull/10518
Let's wait for them to merge and make a new `webpack` version, then we could update the `webpack` version dependency to fix it.

**Comment by Matildevoldsen** [comment]
I've got the same issue and I think it prevents `npm run watch` to watch for the files. However `npm run watch-poll` is the solution for the issue there.

**Comment by parys-github** [comment]
Exact the same issue on my npm :-(

**Comment by HDVinnie** [comment]
@JeffreyWay any update on this?

**Comment by jadamec** [comment]
+ 1

**Comment by G-Rath** [comment]
this still applies :)

**Comment by G-Rath** [comment]
Wrong again `stalebot`

**Comment by G-Rath** [comment]
This is still a problem, and there's "activity" :/

**Comment by HDVinnie** [comment]
I no longer face this issue. Only:
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ yargs-parser                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ laravel-mix [dev]                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ laravel-mix > yargs > yargs-parser                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1500                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Running `Mix Version": "5.0.4"`
